packages/app/src/components/Root/Root.tsx

```diff
@@ -75,6 +75,9 @@ const renderExpandIcon = (expand: boolean, isSecondLevelMenuItem = false) => {
 };
 
 const getMenuItem = (menuItem: ResolvedMenuItem, isNestedMenuItem = false) => {
+  if (!menuItem.title) {
```

Comment:
Can we filter this items in `extractMenuItems`? This makes it much easier to have (an additional) unit test for this.